website-components-playground/src/pages/tabular-modal-page.js

```diff
@@ -50,105 +40,131 @@ const exampleCustomTitleRow = (
-                    onClick={() => {
-                      setIsOpen(false);
-                    }}
-                  />
-                  <TabularModalPage.FormPrimaryButton
-                    onClick={() => undefined}
-                    isDisabled
-                  />
-                  <TabularModalPage.FormDeleteButton
-                    onClick={() => undefined}
-                    isDisabled
-                  />
-                </>
-              }
-              hideControls={values.hideControls}
+              buttonLabel="Open Tabular Modal Page"
+              containerId={containerId}
             >
-              <Text.Body>{values.content}</Text.Body>
-            </TabularModalPage>
+              {({ isOpen, setIsOpen }) => (
+                <TabularModalPage
+                  title={values.title}
+                  isOpen={isOpen}
+                  onClose={() => setIsOpen(false)}
+                  getParentSelector={() =>
+                    document.querySelector(`#${containerId}`)
+                  }
+                  tabControls={
+                    <>
+                      <TabHeader name="tab-one" to="/tab-one">
+                        Tab One
+                      </TabHeader>
+                      <TabHeader to="/tab-two">Tab Two</TabHeader>
+                      <TabHeader name="tab-three" to="/tab-three" isDisabled>
+                        Disabled tab
+                      </TabHeader>
+                    </>
+                  }
+                  customTitleRow={
+                    values.useCustomTitleRow === 'custom' &&
+                    exampleCustomTitleRow
+                  }
+                  formControls={
+                    <>
+                      <TabularModalPage.FormSecondaryButton
+                        onClick={() => {
+                          setIsOpen(false);
+                        }}
+                      />
+                      <TabularModalPage.FormPrimaryButton
+                        onClick={() => undefined}
+                        isDisabled
+                      />
+                      <TabularModalPage.FormDeleteButton
+                        onClick={() => undefined}
+                        isDisabled
+                      />
+                    </>
+                  }
+                  hideControls={values.hideControls}
+                >
+                  <Switch>
+                    <Route
+                      path={[
+                        '/custom-applications/playground/tabular-modal-page/', // production
+                        '/tabular-modal-page/', // development
+                      ]}
+                    >
+                      <Redirect to="/tab-one" />
+                    </Route>
+                    <Route path="/tab-one">
+                      <Text.Body>{values['tab-one-content']}</Text.Body>
+                    </Route>
+                    <Route path="/tab-two">
```

Comment:
These paths should be updated as well.

Reply:
I'm not sure if I got it right but when I changed this to `path={withPrefix("/tab-one")}` it stopped working for production builds

Reply:
I think the path should be like `withPrefix('/tabular-modal-page/tab-one')`